extensions/binding/org.eclipse.smarthome.binding.ntp/META-INF/MANIFEST.MF

```diff
@@ -25,3 +25,4 @@ Import-Package:
  org.eclipse.smarthome.core.types,
  org.slf4j
 Service-Component: OSGI-INF/*.xml
+Bundle-ActivationPolicy: lazy

```

Comment:
Has this only be done, because the Eclipse IDE likes the lazy bundle activation or because there is any real use case?

Has this change been tested that the binding's life cycle is still working?
It should, but in some cases a binding could be activated (AFAIK) earlier then before.

Reply:
Truth to be told, I followed the IDE advices and, in general, am not truly sure if this would have an effect at all. I also saw that it is defined for several of our bindings (e.g. SONOS, Tradfri, Astro, Lifx and DMX), so to me this was a good reason to follow the IDE advice. :) All in all, if we are unsure there is any sense in making this change, I can revert it straightforward.

Reply:
Afaik the IDE's warning could be safely ignored in this case (some say they've even got it all wrong... 😉 ). See also Neil's and Peter's responses [here](https://stackoverflow.com/a/16989358).

Reply:
In such case I will revert this for sure. ;)